engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreSubmitActor.scala

```diff
@@ -21,19 +21,41 @@ final case class WorkflowStoreSubmitActor(store: WorkflowStore, serviceRegistryA
   override def receive = {
     case cmd: SubmitWorkflow =>
       val sndr = sender()
-      storeWorkflowSources(NonEmptyList.of(cmd.source)) foreach { ids =>
-        val id = ids.head
-        registerSubmissionWithMetadataService(id, cmd.source)
-        sndr ! WorkflowSubmittedToStore(id)
-        log.info("Workflow {} submitted.", id)
+
```

Comment:
didn't @Horneth just fiddle w/ this in another PR? or is it the same idea but different thing?